integration-tests/gatsby-source-wordpress/__tests__/__snapshots__/index.js.snap

```diff
@@ -95,7 +95,6 @@ Object {
                 "databaseId": 564,
                 "id": "cG9zdDo1NjQ=",
                 "label": "Activity",
-                "nodeType": "MenuItem",
```

Comment:
These are removed because it was a mistake for them to be Gatsby's schema, this data is only needed on raw node's, not in the schema. For the new wpgql version this was a problem because the nodeType field would now need to be added to all Gatsby node types which I didn't want to do, so I removed it instead.

Reply:
So this is technically breaking? Or wasn't it queryable?

Reply:
It is technically breaking, but in order to support the breaking change in minor version of WPGraphQL, removing it seemed like a better solution. It really wasn't supposed to be in the schema anyway, users can use __typename instead

Reply:
Turns out lots of people use this field, so you were right